personalitytest/personalitytestblock.py

```diff
@@ -78,8 +78,14 @@ def init_quizz(self, data):
         questions = full_quizz['questions']
 
         for question in questions:
-            for answer in question['answers']:
-                answer['categories'] = ''
+            if 'type' not in question:
+                for answer in question['answers']:
+                    answer['categories'] = ''
+            elif question['type'] == 'group':
+                tmp = question['questions']
+                for quest in tmp:
+                    for answer in quest['answers']:
+                        answer['categories'] = ''
```

Comment:
If type is not in question or type is not group, this is a question. 
My condition does not manage this case.
I'll fix it.